account-kit/java/signer-sdk/pom.xml

```diff
@@ -7,9 +7,9 @@
   <description>This is an Alchemy signer SDK</description>
   <url>https://github.com/alchemyplatform/aa-sdk/tree/main/account-kit/java/signer-sdk</url>
 
-  <groupId>io.github.andysim3d</groupId>
+  <groupId>com.alchemy.accountkit</groupId>
   <artifactId>signer-sdk</artifactId>
-  <version>1.1-SNAPSHOT</version>
+  <version>0.0</version>
```

Comment:
Can we set this to `0.0-alpha.{commit_sha}` and add a script to publish it? Would love to be able to track what customers are using